etc/LATISS/primary_hdu.header

```diff
@@ -40,11 +40,11 @@ RAEND   =                      / RA of telescope from AZEND and ELEND
 DECEND  =                      / DEC of telescope from AZEND and ELEND
 ROTPA   =                      / Rotation angle relative to the sky (deg)
 ROTCOORD= 'sky'                / Telescope Rotation Coordinates
-HASTART =                      / [HH:MM:SS] Telescope hour angle at start
+HASTART =                      / [deg] Telescope hour angle at start
```

Comment:
This is decimal hours not degrees.

Reply:
Thanks for noticing, changed to `[decimal hrs]`